lib/plurimath/latex/parse.rb

```diff
@@ -92,8 +92,9 @@ class Parse < Parslet::Parser
       end
 
       rule(:symbol_class_commands) do
-        (slash >> symbols) |
-          (unary >> intermediate_exp.as(:first_value)).as(:unary_functions) |
+        (unary >> intermediate_exp.as(:first_value)).as(:unary_functions) |
+          (str("&#x") >> match["0-9a-fA-F"].repeat >> str(";")).as(:unicode_symbols) |
```

Comment:
Metanorma will only generate hex unicode escapes, but if this gem is to be used in general, it should deal with decimal unicode escapes as well: you can't ban them.